inc/saiport.h

```diff
@@ -1434,6 +1443,26 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_PRIORITY_FLOW_CONTROL_TX,
 
+    /**
+     * @brief Enable/disable port PFC to all bits
+     *
+     * True means enable PFC and false is opposite
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default true
+     */
+    SAI_PORT_ATTR_PRIORITY_FLOW_CONTROL_ALL_BITS,
```

Comment:
no need this attribute.

just add a new enum.

SAI_PORT_PRIORITY_FLOW_CONTROL_MODE_DISABLED around line 192.

Reply:
Done